CHANGELOG.md

```diff
@@ -6,13 +6,17 @@
 - Scaleway Provider (#1643) @Sh4d1
 - Enable azure_private_dns to work with non "AzurePublicCloud" clouds (#1578) @daddonpa
 - Fix typos in documentation @ddymko
+<<<<<<< HEAD
```

Comment:
Please delete this line. Also, please delete lines 17 and 19. Sorry about this.

Reply:
Ahh sorry about it. Fixed